### ISSUE TITLE
Camera not working after a few hours

### ISSUE THREAD

**Issue by benharris2510** [checkra1n/BugTracker#463]
**What device + iOS version are you on?**
iPhone X iOS 13.2.2

**What checkra1n version are you using?**
0.9.5

**What are the steps to reproduce the issue?**
1. Jailbreak
2. Wait around 8 hours
3. Camera stops working 
...

**What do you expect, and what is happening instead?**
There is not input from the front or rear camera, however Face ID still works. It is not just the default Camera app as Snapchat etc has the same problem. 

**Any other info, error logs, screenshots, ...?** No, and to prove camera isn’t working, it allows me to enabled the flashlight at the same time.

**Comment by wm-tbourne** [comment]
Same issue, iPhone X 13.2.2.. After a while starting the camera seems to initiate ok, then it will blur out and freeze. At that point all controls become unresponsive. If I quit and then relaunch the camera app the screen is just black.

**Comment by rylan-silveira** [comment]
I have the same issue, I have to keep re jailbreaking Everyday day or so whenever this occurs. I am using an iPhone 8 Plus , iOS 13.2.2, checkra1n version 0.9.5

**Comment by nullpixel** [comment]
Sounds like a tweak to me, what tweaks do you have installed?

**Comment by rylan-silveira** [comment]
I don’t think so.. why would it take hours for this problem to occur after jailbreaking? If it was a tweak issue it should have happened immediately I feel. Anyway here’s the list

Noctis Neo
Watusi
Apeturizer
Hiddenalbumlock
FloatingDockPlus13

**Comment by benharris2510** [comment]
I tried it for one day without any tweaks installed and the same thing happened. Also got a conspiracy theory about snapchat cos it don’t let me open stories without accessing my camera :D but that’s not important

**Comment by iCrazeiOS** [comment]
camera works in safemode, but does the bug with no tweaks

**Comment by konner1001** [comment]
I second this and I’m on an iP8+ iOS 13.2.2
Edit: found a fix but it’s too soon to say if it’s temporary or not 
Use any terminal app from Cydia and run the command ldrestart

**Comment by rylan-silveira** [comment]
Does this fix it permanently, or do I do this every time the issue occurs?

**Comment by PatrikTheDev** [comment]
Same issue here, the only fix is to run ldrestart. I’m on an iPhone X on 13.3b2 with Checkra1n 0.9.5. It doesn’t work in Safe mode so not tweak related. Started happening with 0.9.5 (0.9.2 was fine)

**Comment by iCrazeiOS** [comment]
happens on all versions

**Comment by PatrikTheDev** [comment]
All versions of iOS or checkra1n? Because it works on 0.9.2 but breaks once I upgrade to 0.9.5.

**Comment by iCrazeiOS** [comment]
checkra1n. im still using 0.9

**Comment by iCrazeiOS** [comment]
and its likely all of iOS 13

**Comment by PatrikTheDev** [comment]
You’re right, still happened after the downgrade. Tho it took much longer.

**Comment by yur1xpp** [comment]
I’m having this issue too. iPhone X 13.3.1, checkra1n 0.9.8.2.
@nullpixel

**Comment by EverlyScott** [comment]
having this issue too... checkra1n 1.0.1, iPhoneOS 13.3.1... seems like a LDRestart works, so I just put that in my control center with CCModules

**Comment by mbquoc** [comment]
X on 14.4.1 and i've try erything i learned on google (ldrestart, reboot userspace...) but still not working...
